delft/sequenceLabelling/preprocess.py

```diff
@@ -363,7 +372,12 @@ def load(cls, file_path):
             variables = json.load(f)
             self = cls()
             for key, val in variables.items():
-                setattr(self, key, val)
+                if key == 'feature_preprocessor' and val is not None:
+                    preprocessor = FeaturesPreprocessor()
+                    preprocessor.__dict__ = val
```

Comment:
perhaps use `__dict__.update` in that case? That seems to be the default implementation of setting a state (see [PEP 307](https://www.python.org/dev/peps/pep-0307/)). Otherwise this could cause issues in the future when you add properties that are not in the JSON.